examples/ssh_example.py

```diff
@@ -5,13 +5,17 @@
 def main():
     scenario = Scenario()
 
-    net = Network("10.0.0.0", "255.255.255.0")
+    net = Network("10.0.0.0", "255.255.255.0", delay="200ms")
 
     node1 = DockerNode('pong', docker_build_dir='./docker/pong')
     node2 = SwitchNode('bridge-1')
     node3 = SSHNode('ping', '10.243.42.11')
-    net.connect(node1, node2, delay='200ms')
-    net.connect(node2, node3, delay='200ms')
+    net.create_channel("channel1")
+    net.create_channel("channel2")
+    net.connect(node1, channel_name="channel1")
+    net.connect(node2, channel_name="channel1")
+    net.connect(node2, channel_name="channel2")
+    net.connect(node3, channel_name="channel2")
```

Comment:
I'd prefer if linking the nodes to channel would be done via an object reference (even if that object was just the channel name, e.g. `channel1 = net.create_channel('channel1')` -> `channel1 == 'channel1'`). This would make the composition less error prone. 

Also, it's tedious to come up with names such as 'channelX' -- I'd prefer if `create_channel` would generate a unique identifier if no `channel_name` was explicitly provided.